lib/cosmos/api/limits_api.rb

```diff
@@ -237,44 +264,34 @@ def set_limits(target_name, packet_name, item_name, red_low, yellow_low, yellow_
     # @return [Hash{String => Array<Array<String, String, String>>]
     def get_limits_groups(scope: $cosmos_scope, token: $cosmos_token)
       authorize(permission: 'tlm', scope: scope, token: token)
-      JSON.parse(Store.instance.hget("#{scope}__cosmos_system", 'limits_groups'))
+      groups = Store.hget("#{scope}__cosmos_system", 'limits_groups')
```

Comment:
No store.   The point is that all these random keys are kept in one place in Models.    I also don't really like the cosmos_system key... Not sure what the better solution is.   Maybe define limits groups in their own primary key, and then keep a different subkey by targetname that defined them?  That would be easier to cleanup, but maybe harder to use?

Reply:
I decided to store the list of limits groups with the target. See what you think